src/Domain/Ingredients/Dto/Ingredient.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Ingredients\Dto;
+
+class Ingredient
+{
+    private string $quantity;
+    private string $measurement;
+    private string $name;
+
+    public function __construct(string $quantity, string $measurement, string $name)
```

Comment:
Is `quantity` a `string` or `int`?

Reply:
Since I extract it from a `string` and also there could be fractions inside like `1 1/2` oz of something, I would like to keep it a string, unless there is a better way to do it